targets/TARGET_STMBLUE/TARGET_BLUENRG2/TARGET_STEVAL_IDB008V2/device/TOOLCHAIN_GCC_ARM/BlueNRG2_old.txt

```diff
@@ -0,0 +1,336 @@
+ /*******************************************************************************
```

Comment:
.txt file? Is this used anywhere? there are few .txt files (linker scripts)

Reply:
@0xc0170 I apologize, it's a commit error. I was using it while refactoring the correct linker script file. I'll eliminate on the next commit.